pkg/queue/cluster_queue_interface.go

```diff
@@ -17,19 +17,29 @@ limitations under the License.
 package queue
 
 import (
+	"context"
 	"fmt"
 
 	"k8s.io/apimachinery/pkg/util/sets"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1alpha1"
 	"sigs.k8s.io/kueue/pkg/workload"
 )
 
+type RequeueReason string
+
+const (
+	RequeueReasonFailedAfterNomination RequeueReason = "FailedAfterNomination"
```

Comment:
Prefer `FailedAfterNomination`, I think it's enough.

Reply:
I am not sure because the `RequeueReasonGeneric` would look strange with the prefix (just `Generic`)